Aliases/wires

```diff
@@ -0,0 +1 @@
+../Formula/geckodriver.rb
```

Comment:
You don't need to add an alias. The information is extracted from `formula_renames.json` and the formula will still be accessible via its old name without the alias.